package.json

```diff
@@ -77,16 +77,16 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^9.1.0",
+    "@angular/animations": "^11.2.5",
     "@angular/cdk": "^8.2.3",
-    "@angular/common": "^9.1.0",
-    "@angular/compiler": "^9.1.0",
-    "@angular/core": "^9.1.0",
+    "@angular/common": "^11.2.5",
+    "@angular/compiler": "^11.2.5",
+    "@angular/core": "^11.2.5",
     "@angular/fire": "5.2.1",
```

Comment:
@angular/fire is now at `6.1.4`

Reply:
Bumping `@angular/fire` to 6.1.4 makes `npm start` fail so I'll postpone that upgrade